lib/semantic_puppet/dependency.rb

```diff
@@ -64,10 +73,12 @@ def query(modules)
     # @param graph [Graph] the root of a dependency graph
     # @return [Array<ModuleRelease>] the list of releases to act on
     def resolve(graph)
+      @module_dependencies, @unsatisfieds = []
```

Comment:
`@unsatisfieds` seems not used

Reply:
That's a typo, I meant to write `@satisfieds`. I wanted to empty the variables before resolving since `SemanticPuppet::Dependency` is a module and its methods can be called multiple times per instance (unlikely, but possible), so the values of `@module_dependencies` and `@satisfieds` might carry over.